requirements.txt

```diff
@@ -7,25 +7,24 @@ django-extensions>=2.2,<3.2
 psycopg2-binary>=2.8,<2.9
 
 # Sandbox
-Pillow==8.1.0
-Whoosh==2.7.4
-django-environ==0.4.5
-django-redis==4.12.1
-pysolr==3.9.0
-redis==3.5.3
-requests==2.25.1
-uWSGI==2.0.19.1
-whitenoise==5.2.0
+Whoosh>=2.7,<2.8
+django-environ>=0.4,<0.5
+django-redis>=4.12,<4.13
+pysolr>=3.9,<4
+redis>=3.5,<3.6
+requests>=2.25,<3
+uWSGI>=2.0.19,<2.1
+whitenoise>=5.2,<5.3
 
 # Linting
 flake8==3.8.4
 flake8-debugger==4.0.0
 isort==5.7.0
 
 # Helpers
-pyprof2calltree==1.4.5
-ipdb==0.13.4
-ipython>=7.12,<7.21
+pyprof2calltree>=1.4,<1.5
+ipdb>=0.13,<0.14
+ipython>=7.12,<8
 
 # Country data
-pycountry>=19.8,<20.8
+pycountry>=19.8,<20
```

Comment:
They do time-based releases (with the version reflecting the release date), with the latest being 20.7.3, made on July 3, 2020. Maybe we should use today's date as the upper limit (i.e. `pycountry>=20.7.3,<21.3.11`)?

Reply:
That would defeat the purpose of having a flexible upper limit?